download_pkgs

```diff
@@ -65,7 +65,10 @@ xargs apt-cache depends --recurse --no-recommends --no-suggests --no-conflicts -
 awk '{ print $1; print $1 ":amd64"; print $1 ":arm64" }' < recursive_depends > recursive_depends_extended
 xargs apt-cache depends --recurse --no-recommends --no-suggests --no-conflicts --no-breaks --no-replaces --no-enhances < recursive_depends_extended | grep '^\w' | sort | uniq > recursive_depends_real
 
+# Exclude pkg:arm64, pkg:amd64, pkg:any
 sed 's/$/:.*/' < "$exclude_patterns" > exclude_extended
+# Exclude pkg (no :arch suffix)
+awk '{ print $1 }' < "$exclude_patterns" >> exclude_extended
```

Comment:
the goal is to have both `<pattern>` and `<pattern>:.*` in `exclude_extended`, right? In that case, can we just do `sed 'p;s/$/:.*/'`?

Reply:
ah nice! 👍🏻true